packages/@aws-cdk/aws-appmesh/test/test.virtual-service.ts

```diff
@@ -122,4 +122,35 @@ export = {
       },
     },
   },
+
+  'When creating a VirtualService': {
+    'with shared service mesh': {
+      'Mesh Owner is the AWS account ID of the account that shared the mesh with your account'(test:Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
```

Comment:
Same comment here - let's explicitly set the account of the Stack to be different than the account of the Mesh.